### ISSUE TITLE
negative value of confidence scores

### ISSUE THREAD

**Issue by mehreenjabeen** [ARM-software/keyword-transformer#5]
Hi,

while running tflite modes from [link](https://github.com/ARM-software/keyword-transformer/tree/master/models_data_v2_12_labels) , I am getting score of label that is predicted as greater than 1 and negative scores for the rest 11 class labels. 
this is one such example score list I got:
[ 4.3287315  -0.24997163 -0.222877   -0.22883172 -0.32111272 -0.31156838 0.27332366 -0.2127423  -0.22332205 -0.3183994  -0.31955314 -0.31245583]

can you explain this.

**Comment by axeber01** [comment]
This is the expected result, since the model returns the unnormalized logits. In order to get confidence scores, you have to apply a softmax function to the output.
